src/main/java/org/skriptlang/skript/bukkit/itemcomponents/equippable/elements/EffEquipCompDispensable.java

```diff
@@ -0,0 +1,61 @@
+package org.skriptlang.skript.bukkit.itemcomponents.equippable.elements;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.*;
+import ch.njol.skript.lang.Effect;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.util.Kleenean;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+import org.skriptlang.skript.bukkit.itemcomponents.equippable.EquippableExperimentSyntax;
+import org.skriptlang.skript.bukkit.itemcomponents.equippable.EquippableWrapper;
+
+@Name("Equippable Component - Dispense")
+@Description("""
+	Whether the item can be dispensed by a dispenser.
+	NOTE: Equippable component elements are experimental. Thus, they are subject to change and may not work as intended.
+	""")
+@Example("allow {_item} to be dispensed")
+@Example("""
+	set {_component} to the equippable component of {_item}
+	prevent {_component} from being dispensed
+	""")
+@RequiredPlugins("Minecraft 1.21.2+")
+@Since("INSERT VERSION")
+public class EffEquipCompDispensable extends Effect implements EquippableExperimentSyntax {
+
+	static {
+		Skript.registerEffect(EffEquipCompDispensable.class,
+			"(allow|force) %equippablecomponents% to be dispensed",
```

Comment:
Changing my mind on `force` here. Doesn't convey the right meaning in this context.